redis.go

```
@@ -40,6 +43,8 @@ func NewRedisCache(endpoint string, opts ...option) (*RedisCache, error) {
 			// 	splitChunkSize = o.value.(int64)
 		case optionNameDebugWriter:
 			w = o.value.(io.Writer)
+		case optionNameScanCount:
+			scanCount = o.value.(int64)
```

We need to validate option value is not a negative integer. if the value is less than 0, we should return an error.

---

I fixed it to validate received option.